integration/operator_test.go

```diff
@@ -168,8 +168,11 @@ func TestMain(m *testing.M) {
 
 	if os.Getenv("KEEP_RESOURCES") != "true" {
 		f.DeleteGuestCluster()
-		operatorTearDown()
-		f.TearDown()
+		// only do full teardown when not on CI
+		if os.Getenv("CIRCLECI") != "true" {
+			operatorTearDown()
+			f.TearDown()
+		}
```

Comment:
What do those functions do?

Reply:
They remove components installed for the build https://github.com/giantswarm/aws-operator/blob/master/integration/operator_test.go#L311-L316 and https://github.com/giantswarm/e2e-harness/blob/master/pkg/framework/framework.go#L211-L214 from minikube/localkube.

Reply:
So this basically means we don't clean the minikube ran inside circleci?

Reply:
Exactly, we don't care about the state of the circleCI machine once the build is done, it can be even useful to keep these components for debugging.